frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -334,7 +340,7 @@ export default class ArenaContestList extends Vue {
   currentOrder: ContestOrder = ContestOrder.None;
   currentFilterBySignedUp: boolean = false;
   currentFilterByRecommended: boolean = false;
-  currentPageSize: number = 10;
+  busy: boolean = false;
```

Comment:
```suggestion
  refreshing: boolean = false;
```

creo que esto es más cercano a describir la realidad de lo que está ocurriendo